src/script/mls/MLSMigration/migrationStatus.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {FeatureMLSMigration, FeatureStatus} from '@wireapp/api-client/lib/team';
+
+const hasMigrationStartTimeArrived = (mlsMigrationFeature: FeatureMLSMigration): boolean => {
+  if (mlsMigrationFeature.status === FeatureStatus.DISABLED) {
+    return false;
+  }
+
+  const startDateISO = mlsMigrationFeature.config.startTime;
+  const startTime = (startDateISO && Date.parse(startDateISO)) || Infinity;
+  const hasStartTimeArrived = Date.now() >= startTime;
+
+  return hasStartTimeArrived;
```

Comment:
You don't need to declare new variable ;) It also is readable :)

```suggestion
  return Date.now() >= startTime;
```